packages/components/src/components/Election/Schedule.tsx

```diff
@@ -0,0 +1,29 @@
+import { HeadingProps } from '@chakra-ui/layout'
+import { chakra, forwardRef, useMultiStyleConfig } from '@chakra-ui/system'
+import { format as dformat } from 'date-fns'
+
+import { useElection } from './Election'
+
+export type ElectionScheduleProps = HeadingProps & {
+  format?: string
+}
+
+export const ElectionSchedule = forwardRef<ElectionScheduleProps, 'h2'>(({ format, ...rest }, ref) => {
+  const styles = useMultiStyleConfig('ElectionSchedule', rest)
+  const { election } = useElection()
+
+  let f = format
+  if (!f) {
+    f = 'd-L-y HH:mm'
```

Comment:
Is not easier to just do something like:

```
export const ElectionSchedule = forwardRef<ElectionScheduleProps, 'h2'>(({ format = 'd-L-y HH:mm', ...rest }, ref) => {
```

Reply:
Maybe 🙃 I wasn't aware we could define default values in object-spread-like arguments.